src/pages/[platform]/build-a-backend/troubleshooting/library-not-configured/index.mdx

```diff
@@ -0,0 +1,149 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Library Not Configured',
+  description: 'Addressing missing configuration or NoCredentials error messages',
+  platforms: [
+    'angular',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'vue'
+  ],
+  canonicalObjects: [
+    {
+      platforms: [
+        'angular',
+        'react',
+        'react-native',
+        'vue',
+        'nextjs',
+        'javascript'
+      ],
+      canonicalPath: '/angular/build-a-backend/troubleshooting/migrate-from-javascript-v5-to-v6/'
+    }
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+If you are running into a missing configuration or `NoCredentials` error message and have called `Amplify.configure` in your project, your Amplify API is most likely being called before `Amplify.configure`. This can happen in a few different ways. Below are three possibilities you can check to troubleshoot this issue.
+
+## Call `Amplify.configure` in the root of your project
+
+Make sure you are calling `Amplify.configure` in the root file of your project. The root file of your app may be different depending on your frontend framework. The current default for some common frameworks are listed below (if you are not using TypeScript the `ts` and `tsx` extensions would be `js` and `jsx`):
+
+* Vue.js: **src/main.ts**
+* React: **src/main.tsx**
+* Angular: **src/main.ts**
+* Next.js Page Router: **pages/_app.tsx** or **src/pages/_app.tsx**
```

Comment:
You could also add Nuxt to this list.  You should run the configuration in the `app.vue` [file](https://nuxt.com/docs/guide/directory-structure/app) in the root. Or in a plugins file, as [recommended here](https://docs.amplify.aws/react/build-a-backend/server-side-rendering/nuxt/#pageMain).

Reply:
Oh nice, thanks!